tests/AppBundle/AppTestTrait.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Tests\AppBundle;
+
+use Doctrine\ORM\Tools\SchemaTool;
+
+trait AppTestTrait
```

Comment:
in this case it's not good practice to write trait. Main problems from my point view:
1) you use `$this->em` but what if trait will be included in class without `em`?
2) you use `postFixtureSetup` but what if trait will be included in class without this method?